download-dns-challenge-4.sh

```diff
@@ -0,0 +1,497 @@
+    datasets_fullband/german_speech_015_NA_NA.tar.bz2
+    datasets_fullband/german_speech_016_NA_NA.tar.bz2
+    datasets_fullband/german_speech_017_NA_NA.tar.bz2
+    datasets_fullband/german_speech_018_NA_NA.tar.bz2
+    datasets_fullband/german_speech_019_NA_NA.tar.bz2
+    datasets_fullband/german_speech_020_NA_NA.tar.bz2
+    datasets_fullband/german_speech_021_NA_NA.tar.bz2
+    datasets_fullband/german_speech_022_NA_NA.tar.bz2
+    datasets_fullband/german_speech_023_NA_NA.tar.bz2
+    datasets_fullband/german_speech_024_NA_NA.tar.bz2
+    datasets_fullband/german_speech_025_NA_NA.tar.bz2
+    datasets_fullband/german_speech_026_NA_NA.tar.bz2
+    datasets_fullband/german_speech_027_NA_NA.tar.bz2
+    datasets_fullband/german_speech_028_NA_NA.tar.bz2
+    datasets_fullband/german_speech_029_NA_NA.tar.bz2
+    datasets_fullband/german_speech_030_NA_NA.tar.bz2
+    datasets_fullband/german_speech_031_NA_NA.tar.bz2
+    datasets_fullband/german_speech_032_NA_NA.tar.bz2
+    datasets_fullband/german_speech_033_NA_NA.tar.bz2
+    datasets_fullband/german_speech_034_NA_NA.tar.bz2
+    datasets_fullband/german_speech_035_NA_NA.tar.bz2
+    datasets_fullband/german_speech_036_NA_NA.tar.bz2
+    datasets_fullband/german_speech_037_NA_NA.tar.bz2
+    datasets_fullband/german_speech_038_NA_NA.tar.bz2
+    datasets_fullband/german_speech_039_NA_NA.tar.bz2
+    datasets_fullband/german_speech_040_NA_NA.tar.bz2
+    datasets_fullband/german_speech_041_NA_NA.tar.bz2
+    datasets_fullband/german_speech_042_NA_NA.tar.bz2
+    datasets_fullband/german_speech_043_NA_NA.tar.bz2
+    datasets_fullband/german_speech_044_NA_NA.tar.bz2
+    datasets_fullband/german_speech_045_NA_NA.tar.bz2
+    datasets_fullband/german_speech_046_NA_NA.tar.bz2
+    datasets_fullband/german_speech_047_NA_NA.tar.bz2
+    datasets_fullband/german_speech_048_NA_NA.tar.bz2
+    datasets_fullband/german_speech_049_NA_NA.tar.bz2
+    datasets_fullband/german_speech_050_NA_NA.tar.bz2
+    datasets_fullband/german_speech_051_NA_NA.tar.bz2
+    datasets_fullband/german_speech_052_NA_NA.tar.bz2
+    datasets_fullband/german_speech_053_NA_NA.tar.bz2
+    datasets_fullband/german_speech_054_NA_NA.tar.bz2
+    datasets_fullband/german_speech_055_NA_NA.tar.bz2
+    datasets_fullband/german_speech_056_NA_NA.tar.bz2
+    datasets_fullband/german_speech_057_NA_NA.tar.bz2
+    datasets_fullband/german_speech_058_NA_NA.tar.bz2
+    datasets_fullband/german_speech_059_NA_NA.tar.bz2
+    datasets_fullband/german_speech_060_NA_NA.tar.bz2
+    datasets_fullband/german_speech_061_NA_NA.tar.bz2
+    datasets_fullband/german_speech_062_NA_NA.tar.bz2
+    datasets_fullband/german_speech_063_NA_NA.tar.bz2
+    datasets_fullband/german_speech_064_NA_NA.tar.bz2
+    datasets_fullband/german_speech_065_NA_NA.tar.bz2
+    datasets_fullband/german_speech_066_NA_NA.tar.bz2
+    datasets_fullband/german_speech_067_NA_NA.tar.bz2
+    datasets_fullband/german_speech_068_NA_NA.tar.bz2
```

Comment:
Everything looks good except for the files with NA_NA instead of actual scores. Let's fix that and I can merge the PR